Eslint and build fixes

Gijsdeman opened the pull request:
# Description
Align project with new eslint rules, upgrade dependencies, fix build and dotenv issues.

## Types of changes
<!-- What types of changes does your code introduce? Remove all the items that do not apply: -->
- Bug fix _(non-breaking change which fixes an issue)_
- Style _(Change that do not affect the functionality of the code)_
- CI/CD _(Changes to the CI/CD configuration)_

Gijsdeman committed:
chore: move eslint config

Gijsdeman committed:
style: eslint fixes

Gijsdeman committed:
ci: group dependabot PRs

dependabot[bot] committed:
chore(deps): update dependencies

Bump mysql2 from 3.12.0 to 3.13.0
Bump pg from 8.13.3 to 8.14.1
chore(deps-dev): bump the development_minors_and_patches group across 1 directory with 5 updates
chore(deps): bump the production_patches group across 1 directory with 4 updates

Signed-off-by: dependabot[bot] <support@github.com>

Gijsdeman committed:
fix: dockerfile, mime and dotenv

LuukBlankenstijn commented on package.json:
```diff
@@ -11,14 +11,14 @@
     "gen-client": "nswag openapi2tsclient /input:src/public/swagger.json /output:../parelpracht-client/src/clients/server.generated.ts /ServiceHost:.",
```
TSOA seems to put swagger.json and routes.ts in /build/ instead of /src/public/. This makes it the gen-client command does not run..

LuukBlankenstijn reviewed (approved):
It is not exactly clear to me what the relation between this PR and #42 is. Does that one need to be merged first?

Yoronex committed:
chore(deps): update dependencies (#71)

* chore(deps): update dependencies

* chore(deps): remove double dependency

dependabot[bot] committed:
chore(deps): bump the production_patches group across 1 directory with 4 updates (#74)

Bumps the production_patches group with 4 updates in the / directory: [node-cron](https://github.com/merencia/node-cron), [pg](https://github.com/brianc/node-postgres/tree/HEAD/packages/pg), [strong-error-handler](https://github.com/loopbackio/strong-error-handler) and [typeorm](https://github.com/typeorm/typeorm).


Updates `node-cron` from 4.1.0 to 4.1.1
- [Release notes](https://github.com/merencia/node-cron/releases)
- [Commits](https://github.com/merencia/node-cron/compare/v4.1.0...v4.1.1)

Updates `pg` from 8.16.0 to 8.16.2
- [Changelog](https://github.com/brianc/node-postgres/blob/master/CHANGELOG.md)
- [Commits](https://github.com/brianc/node-postgres/commits/pg@8.16.2/packages/pg)

Updates `strong-error-handler` from 5.0.19 to 5.0.20
- [Release notes](https://github.com/loopbackio/strong-error-handler/releases)
- [Changelog](https://github.com/loopbackio/strong-error-handler/blob/master/CHANGES.md)
- [Commits](https://github.com/loopbackio/strong-error-handler/compare/v5.0.19...v5.0.20)

Updates `typeorm` from 0.3.24 to 0.3.25
- [Release notes](https://github.com/typeorm/typeorm/releases)
- [Changelog](https://github.com/typeorm/typeorm/blob/master/CHANGELOG.md)
- [Commits](https://github.com/typeorm/typeorm/compare/0.3.24...0.3.25)

---
updated-dependencies:
- dependency-name: node-cron
  dependency-version: 4.1.1
  dependency-type: direct:production
  update-type: version-update:semver-patch
  dependency-group: production_patches
- dependency-name: pg
  dependency-version: 8.16.2
  dependency-type: direct:production
  update-type: version-update:semver-patch
  dependency-group: production_patches
- dependency-name: strong-error-handler
  dependency-version: 5.0.20
  dependency-type: direct:production
  update-type: version-update:semver-patch
  dependency-group: production_patches
- dependency-name: typeorm
  dependency-version: 0.3.25
  dependency-type: direct:production
  update-type: version-update:semver-patch
  dependency-group: production_patches
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump the development_minors_and_patches group across 1 directory with 6 updates (#75)

---
updated-dependencies:
- dependency-name: "@types/jsonwebtoken"
  dependency-version: 9.0.10
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development_minors_and_patches
- dependency-name: "@types/lodash"
  dependency-version: 4.17.18
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development_minors_and_patches
- dependency-name: "@types/node"
  dependency-version: 22.15.32
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development_minors_and_patches
- dependency-name: "@types/validator"
  dependency-version: 13.15.2
  dependency-type: direct:development
  update-type: version-update:semver-patch
  dependency-group: development_minors_and_patches
- dependency-name: concurrently
  dependency-version: 9.2.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development_minors_and_patches
- dependency-name: eslint
  dependency-version: 9.29.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
  dependency-group: development_minors_and_patches
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Yoronex committed:
docs: update README.md
